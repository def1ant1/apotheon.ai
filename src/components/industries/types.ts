import type { IndustryIconSlug } from '../../content/industries/iconSlugs';

/**
 * Shared data contracts for industry components. We replicate the schema shape here to
 * avoid pulling runtime dependencies on `astro:content` inside component tests while
 * keeping editors aligned with the frontmatter contract.
 */
export interface IndustryHeroContent {
  eyebrow: string;
  headline: string;
  copy: string;
  icon: IndustryIconSlug;
}

export interface IndustryPressure {
  title: string;
  description: string;
  metric?: string;
}

export interface IndustryComplianceHighlight {
  framework: string;
  highlight: string;
  evidence?: string;
}

export interface IndustrySolutionReference {
  slug: string;
  positioning: string;
  outcome?: string;
}

export interface IndustrySolutionSummary {
  slug: string;
  data: {
    title: string;
  };
}

export interface IndustryUseCase {
  title: string;
  persona: string;
  narrative: string;
  automationLevel?: string;
}

export interface IndustryCta {
  label: string;
  href: string;
  description?: string;
  ariaLabel?: string;
}

export interface IndustryCtaGroup {
  demo: IndustryCta;
  whitepaper: IndustryCta;
}
