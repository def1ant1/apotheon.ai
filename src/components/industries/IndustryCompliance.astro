---
import type { IndustryComplianceHighlight } from './types';

interface Props {
  complianceHighlights: IndustryComplianceHighlight[];
}

const { complianceHighlights } = Astro.props as Props;
const sectionId = 'industry-compliance';
---

<section
  id={sectionId}
  aria-labelledby={`${sectionId}-heading`}
  class="space-y-6 rounded-3xl border border-slate-800 bg-slate-950/60 p-8 shadow-lg ring-1 ring-slate-900/60"
  data-analytics-region="industry-compliance"
>
  <!--
    Compliance messaging is mission critical for enterprise deals. We highlight the
    framework first, then narrate the alignment, and close with optional evidence.
    The server-rendered markup keeps everything crawlable for security scanners and
    search crawlers while avoiding hydration.
  -->
  <header class="space-y-2">
    <h2 id={`${sectionId}-heading`} class="text-2xl font-semibold text-white md:text-3xl">
      Compliance proof points
    </h2>
    <p class="text-base text-slate-300">
      Procurement, security, and risk teams can trace each control back to the supporting
      attestation package.
    </p>
  </header>

  <ul class="grid gap-6 md:grid-cols-3" data-testid="industry-compliance-list">
    {
      complianceHighlights.map((highlight) => (
        <li class="rounded-2xl border border-slate-800/80 bg-slate-900/50 p-6 shadow-inner">
          <h3 class="text-lg font-semibold text-white">{highlight.framework}</h3>
          <p class="mt-3 text-sm text-slate-300">{highlight.highlight}</p>
          {highlight.evidence ? (
            <p class="mt-3 text-xs font-semibold uppercase tracking-[0.3rem] text-indigo-300">
              Evidence: {highlight.evidence}
            </p>
          ) : null}
        </li>
      ))
    }
  </ul>
</section>
