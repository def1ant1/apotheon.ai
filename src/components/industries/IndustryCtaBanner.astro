---
import type { IndustryCtaGroup } from './types';

interface Props {
  ctas: IndustryCtaGroup;
}

const { ctas } = Astro.props as Props;
const sectionId = 'industry-cta-banner';
---

<section
  id={sectionId}
  aria-labelledby={`${sectionId}-heading`}
  class="space-y-6 rounded-3xl border border-indigo-500/50 bg-indigo-500/10 p-8 shadow-lg ring-1 ring-indigo-500/30"
  data-analytics-region="industry-cta"
>
  <!--
    CTA banner keeps both demo and whitepaper options visible without requiring
    client-side toggles. Buttons remain server-rendered anchors so analytics can
    attribute conversions without extra instrumentation.
  -->
  <header class="space-y-2">
    <h2 id={`${sectionId}-heading`} class="text-2xl font-semibold text-white md:text-3xl">
      Continue the conversation
    </h2>
    <p class="text-base text-indigo-100">
      Choose between a guided platform walkthrough or self-serve research packetâ€”both reinforce the
      compliance story above.
    </p>
  </header>

  <div class="flex flex-col gap-4 md:flex-row">
    <a
      class="inline-flex items-center justify-center gap-2 rounded-xl border border-indigo-400/60 bg-indigo-500/30 px-6 py-3 text-center text-base font-semibold text-white shadow-lg transition hover:bg-indigo-500/40 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-4 focus-visible:outline-white"
      href={ctas.demo.href}
      aria-label={ctas.demo.ariaLabel ?? `${ctas.demo.label} - demo CTA`}
      data-cta="demo"
    >
      {ctas.demo.label}
    </a>
    <a
      class="inline-flex items-center justify-center gap-2 rounded-xl border border-slate-200/20 bg-slate-900/40 px-6 py-3 text-center text-base font-semibold text-indigo-100 shadow-lg transition hover:border-indigo-300/50 hover:text-white focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-4 focus-visible:outline-indigo-200"
      href={ctas.whitepaper.href}
      aria-label={ctas.whitepaper.ariaLabel ?? `${ctas.whitepaper.label} - whitepaper CTA`}
      data-cta="whitepaper"
    >
      {ctas.whitepaper.label}
    </a>
  </div>

  <div class="space-y-2 text-sm text-indigo-100/90">
    {ctas.demo.description ? <p>{ctas.demo.description}</p> : null}
    {ctas.whitepaper.description ? <p>{ctas.whitepaper.description}</p> : null}
  </div>
</section>
