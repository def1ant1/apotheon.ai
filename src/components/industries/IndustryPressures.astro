---
import type { IndustryPressure } from './types';

interface Props {
  pressures: IndustryPressure[];
}

const { pressures } = Astro.props as Props;
const sectionId = 'industry-pressures';
---

<section
  id={sectionId}
  aria-labelledby={`${sectionId}-heading`}
  class="space-y-6 rounded-3xl border border-slate-800 bg-slate-950/60 p-8 shadow-lg ring-1 ring-slate-900/60"
  data-analytics-region="industry-pressures"
>
  <!--
    Pressure cards ship server-rendered definition lists to keep reading order and
    SEO semantics intact. No client-side hydration is required because the content
    resolves entirely from structured frontmatter.
  -->
  <header class="space-y-2">
    <h2 id={`${sectionId}-heading`} class="text-2xl font-semibold text-white md:text-3xl">
      Sector pressures we neutralize
    </h2>
    <p class="text-base text-slate-300">
      Executive sponsors consistently cite these systemic blockers when evaluating transformation
      programs.
    </p>
  </header>

  <dl class="grid gap-6 md:grid-cols-3" data-testid="industry-pressures-list">
    {
      pressures.map((pressure) => (
        <div class="rounded-2xl border border-slate-800/80 bg-slate-900/50 p-6 shadow-inner">
          <dt class="text-lg font-semibold text-white">{pressure.title}</dt>
          <dd class="mt-3 space-y-2 text-sm text-slate-300">
            <p>{pressure.description}</p>
            {pressure.metric ? (
              <p class="font-semibold text-indigo-200" data-metric>
                {pressure.metric}
              </p>
            ) : null}
          </dd>
        </div>
      ))
    }
  </dl>
</section>
