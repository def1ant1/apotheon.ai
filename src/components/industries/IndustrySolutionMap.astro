---
import type { IndustrySolutionReference, IndustrySolutionSummary } from './types';

interface Props {
  solutionMap: IndustrySolutionReference[];
  solutions: IndustrySolutionSummary[];
}

const { solutionMap, solutions } = Astro.props as Props;
const sectionId = 'industry-solution-map';
type SolutionSlug = IndustrySolutionSummary['slug'];

const solutionIndex = new Map<SolutionSlug, IndustrySolutionSummary>(
  solutions.map((solution) => [solution.slug, solution]),
);
---

<section
  id={sectionId}
  aria-labelledby={`${sectionId}-heading`}
  class="space-y-6 rounded-3xl border border-slate-800 bg-slate-950/60 p-8 shadow-lg ring-1 ring-slate-900/60"
  data-analytics-region="industry-solution-map"
>
  <!--
    Solution references stay server-rendered and auto-link into the solutions collection.
    We compute hrefs using the canonical slug so editors never have to duplicate URLs
    or worry about stale navigation paths.
  -->
  <header class="space-y-2">
    <h2 id={`${sectionId}-heading`} class="text-2xl font-semibold text-white md:text-3xl">
      Platform capabilities aligned to the mission
    </h2>
    <p class="text-base text-slate-300">
      Each recommendation cross-links to deep dives managed by the solutions team.
    </p>
  </header>

  <ul class="grid gap-6 md:grid-cols-3" data-testid="industry-solution-list">
    {
      solutionMap.map((item) => {
        const solutionSlug = item.slug as SolutionSlug;
        const solution = solutionIndex.get(solutionSlug);
        const href = `/solutions/${solutionSlug}/`;
        return (
          <li
            class="rounded-2xl border border-slate-800/80 bg-slate-900/50 p-6 shadow-inner"
            data-solution-slug={item.slug}
          >
            <div class="space-y-2">
              <p class="text-xs uppercase tracking-[0.3rem] text-indigo-300">Mapped solution</p>
              <h3 class="text-lg font-semibold text-white">
                {solution?.data.title ?? item.slug.replace(/-/g, ' ')}
              </h3>
            </div>
            <p class="mt-3 text-sm text-slate-300">{item.positioning}</p>
            {item.outcome ? (
              <p class="mt-3 text-sm font-semibold text-indigo-200">{item.outcome}</p>
            ) : null}
            <a
              class="mt-4 inline-flex items-center gap-2 text-sm font-semibold text-indigo-200 transition hover:text-indigo-100"
              href={href}
              aria-label={`Explore the ${solution?.data.title ?? item.slug} solution`}
            >
              Explore solution
              <span aria-hidden class="text-lg">
                â†’
              </span>
            </a>
          </li>
        );
      })
    }
  </ul>
</section>
