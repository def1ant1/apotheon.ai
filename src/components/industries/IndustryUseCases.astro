---
import type { IndustryUseCase } from './types';

interface Props {
  useCases: IndustryUseCase[];
}

const { useCases } = Astro.props as Props;
const sectionId = 'industry-use-cases';
---

<section
  id={sectionId}
  aria-labelledby={`${sectionId}-heading`}
  class="space-y-6 rounded-3xl border border-slate-800 bg-slate-950/60 p-8 shadow-lg ring-1 ring-slate-900/60"
  data-analytics-region="industry-use-cases"
>
  <!--
    Use cases render as a semantic definition list so personas and narratives stay
    linked for screen readers. No interactive controls are required, preserving
    the hydration-free philosophy that keeps the page lightweight.
  -->
  <header class="space-y-2">
    <h2 id={`${sectionId}-heading`} class="text-2xl font-semibold text-white md:text-3xl">
      Targeted use cases for cross-functional teams
    </h2>
    <p class="text-base text-slate-300">
      Designed to speak directly to the operators responsible for launching, governing, and scaling
      the platform.
    </p>
  </header>

  <dl class="grid gap-6 md:grid-cols-3" data-testid="industry-use-case-list">
    {
      useCases.map((useCase) => (
        <div class="rounded-2xl border border-slate-800/80 bg-slate-900/50 p-6 shadow-inner">
          <dt class="space-y-1">
            <p class="text-xs font-semibold uppercase tracking-[0.3rem] text-indigo-300">
              {useCase.persona}
            </p>
            <h3 class="text-lg font-semibold text-white">{useCase.title}</h3>
          </dt>
          <dd class="mt-3 space-y-2 text-sm text-slate-300">
            <p>{useCase.narrative}</p>
            {useCase.automationLevel ? (
              <p class="font-semibold text-indigo-200">
                Automation level: {useCase.automationLevel}
              </p>
            ) : null}
          </dd>
        </div>
      ))
    }
  </dl>
</section>
