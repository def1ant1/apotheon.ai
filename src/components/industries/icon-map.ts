import {
  IndustryEnergyIcon,
  IndustryFinanceIcon,
  IndustryGovernmentIcon,
  IndustryHealthcareIcon,
  IndustryIntelligenceIcon,
  IndustryLawIcon,
  IndustryManufacturingIcon,
  IndustryMilitaryIcon,
  IndustryPublicSectorIcon,
  IndustryTransportIcon,
} from '../icons';

import type { IndustryIconSlug } from '../../content/industries/iconSlugs';
import type { ComponentType, SVGProps } from 'react';

export type IndustryIconComponent = ComponentType<SVGProps<SVGSVGElement>>;

const iconMap: Record<IndustryIconSlug, IndustryIconComponent> = {
  finance: IndustryFinanceIcon,
  healthcare: IndustryHealthcareIcon,
  'public-sector': IndustryPublicSectorIcon,
  energy: IndustryEnergyIcon,
  manufacturing: IndustryManufacturingIcon,
  transport: IndustryTransportIcon,
  law: IndustryLawIcon,
  government: IndustryGovernmentIcon,
  military: IndustryMilitaryIcon,
  intelligence: IndustryIntelligenceIcon,
};

export function resolveIndustryIcon(slug: IndustryIconSlug): IndustryIconComponent {
  return iconMap[slug];
}
