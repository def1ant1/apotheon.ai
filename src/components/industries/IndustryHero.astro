---
import { resolveIndustryIcon } from './icon-map';

import type { IndustryHeroContent } from './types';

interface Props {
  hero: IndustryHeroContent;
  title: string;
}

const { hero, title } = Astro.props as Props;
const HeroIcon = resolveIndustryIcon(hero.icon);
const headingId = 'industry-hero-heading';
const descriptionId = 'industry-hero-description';
---

<header
  class="relative isolate overflow-hidden rounded-3xl border border-slate-800 bg-slate-950/70 p-10 shadow-xl ring-1 ring-slate-900/60"
  aria-labelledby={headingId}
  aria-describedby={descriptionId}
  data-analytics-region="industry-hero"
>
  <!--
    Accessibility note: the hero uses a semantic <header> landmark with aria hooks so
    assistive technologies reliably announce the sector headline before scanning the
    rest of the narrative. Copy and buttons remain server-rendered to avoid client
    hydration and keep content indexable.
  -->
  <div class="flex flex-col gap-8 md:flex-row md:items-center md:justify-between">
    <div class="max-w-3xl space-y-4">
      <p class="text-xs font-semibold uppercase tracking-[0.4rem] text-indigo-300">
        {hero.eyebrow}
      </p>
      <h1 id={headingId} class="text-4xl font-semibold text-white md:text-5xl">
        {hero.headline}
      </h1>
      <p id={descriptionId} class="text-lg leading-relaxed text-slate-300 md:text-xl">
        {hero.copy}
      </p>
    </div>
    <div
      class="flex h-24 w-24 shrink-0 items-center justify-center rounded-2xl border border-indigo-500/40 bg-indigo-500/10"
    >
      <!-- Icon is rendered server-side to keep the layout hydration-free while still
           exposing decorative affordances for sighted users. -->
      <HeroIcon className="h-12 w-12 text-indigo-200" aria-hidden="true" focusable="false" />
      <span class="sr-only">{title} industry icon</span>
    </div>
  </div>
</header>
