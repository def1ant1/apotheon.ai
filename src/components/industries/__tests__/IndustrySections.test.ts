import { describe, expect, it } from 'vitest';

import { INDUSTRY_ICON_SLUGS } from '../../../content/industries/iconSlugs';
import { resolveIndustryIcon } from '../icon-map';

import type {
  IndustryComplianceHighlight,
  IndustryCtaGroup,
  IndustryHeroContent,
  IndustryPressure,
  IndustrySolutionReference,
  IndustrySolutionSummary,
  IndustryUseCase,
} from '../types';

describe('Industry section view models', () => {
  const hero: IndustryHeroContent = {
    eyebrow: 'Regulated Industries',
    headline: 'Mission ready intelligence',
    copy: 'Structured playbooks keep executives aligned on compliance, automation, and go-live velocity.',
    icon: 'finance',
  } as const;

  const pressures: IndustryPressure[] = [
    {
      title: 'Siloed telemetry inflates response times',
      description:
        'Teams lose hours stitching together telemetry across finance, risk, and operations dashboards.',
      metric: '52% of operators cite duplicate investigations as a top risk.',
    },
    {
      title: 'Manual attestations delay procurement',
      description:
        'Security teams chase spreadsheet attestations instead of working from shared system-of-record evidence.',
      metric: '8+ hours burned each week summarizing control status.',
    },
    {
      title: 'Automation programs stall on compliance reviews',
      description:
        'Program leads hesitate to ship new flows without explicit policy guardrails baked into orchestrations.',
    },
  ] as const;

  const complianceHighlights: IndustryComplianceHighlight[] = [
    {
      framework: 'SOC 2 Type II + ISO 27001',
      highlight:
        'Shared control inheritance packages accelerate third-party assessments across global teams.',
      evidence: 'SOC 2 Type II report and ISO certificates in vendor portal.',
    },
  ] as const;

  const solutionMap: IndustrySolutionReference[] = [
    {
      slug: 'atlas',
      positioning: 'Centralizes telemetry and governed analytics for financial services teams.',
      outcome: 'Cuts investigation prep time in half.',
    },
    {
      slug: 'automation-studio',
      positioning:
        'Delivers policy-driven automations that pass internal risk reviews without manual rewrites.',
    },
  ] as const;

  const useCases: IndustryUseCase[] = [
    {
      title: 'Unified control room',
      persona: 'Operations executives',
      narrative:
        'Blend telemetry, approvals, and audits into a single mission control experience for stakeholders.',
      automationLevel: 'Automated alerts with manual approvals.',
    },
    {
      title: 'Evidence automation',
      persona: 'Risk and compliance leads',
      narrative:
        'Auto-generate attestations, POA&Ms, and regulator-ready audit packages with reusable templates.',
    },
  ] as const;

  const ctas: IndustryCtaGroup = {
    demo: {
      label: 'Request a tailored demo',
      href: '/about/contact/?flow=demo&vertical=finance',
      description: 'Partner with platform specialists to scope governed telemetry patterns.',
    },
    whitepaper: {
      label: 'Download the compliance brief',
      href: '/assets/whitepapers/sample.pdf',
      description: 'Review the audit-ready architecture and reporting approach.',
    },
  } as const;

  const solutionEntries: IndustrySolutionSummary[] = [
    {
      slug: 'atlas',
      data: { title: 'Atlas Data Fabric' },
    },
    {
      slug: 'automation-studio',
      data: { title: 'Automation Studio' },
    },
  ] as const;

  it('snapshots the hero payload', () => {
    expect({ hero, title: 'Finance Intelligence' }).toMatchSnapshot();
  });

  it('snapshots pressures and metrics', () => {
    expect(pressures).toMatchSnapshot();
  });

  it('snapshots mapped solutions and validates linked titles', () => {
    const resolvedSolutions = solutionMap.map((item) => {
      const match = solutionEntries.find((entry) => entry.slug === item.slug);
      return {
        slug: item.slug,
        title: match?.data.title ?? item.slug.replace(/-/g, ' '),
        href: `/solutions/${item.slug}/`,
        positioning: item.positioning,
        outcome: item.outcome ?? null,
      };
    });

    expect(resolvedSolutions).toMatchSnapshot();
    expect(resolvedSolutions.map((entry) => entry.title)).toEqual([
      'Atlas Data Fabric',
      'Automation Studio',
    ]);
  });

  it('snapshots persona use cases', () => {
    expect(useCases).toMatchSnapshot();
  });

  it('snapshots compliance highlights', () => {
    expect(complianceHighlights).toMatchSnapshot();
  });

  it('snapshots CTA configuration', () => {
    expect(ctas).toMatchSnapshot();
  });

  it('tracks the registered industry icon slugs for schema parity', () => {
    expect(INDUSTRY_ICON_SLUGS).toMatchSnapshot('industry-icon-slugs');
  });

  it.each(INDUSTRY_ICON_SLUGS)(
    'resolves an icon component for %s',
    (slug: (typeof INDUSTRY_ICON_SLUGS)[number]) => {
      /**
       * The icon resolver must always return a renderable component. Some of our
       * generated icons use `forwardRef`, which returns a React element factory
       * object with a `render` method instead of a bare function. We normalize the
       * assertion to cover both patterns without coupling the test to React
       * internals.
       */
      const IconComponent = resolveIndustryIcon(slug);
      expect(IconComponent).toBeTruthy();
      if (typeof IconComponent === 'object') {
        expect('render' in IconComponent).toBe(true);
      } else {
        expect(typeof IconComponent).toBe('function');
      }
    },
  );
});
