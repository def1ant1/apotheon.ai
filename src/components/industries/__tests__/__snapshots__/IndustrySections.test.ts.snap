// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Industry section view models > snapshots CTA configuration 1`] = `
{
  "demo": {
    "description": "Partner with platform specialists to scope governed telemetry patterns.",
    "href": "/about/contact/?flow=demo&vertical=finance",
    "label": "Request a tailored demo",
  },
  "whitepaper": {
    "description": "Review the audit-ready architecture and reporting approach.",
    "href": "/assets/whitepapers/sample.pdf",
    "label": "Download the compliance brief",
  },
}
`;

exports[`Industry section view models > snapshots compliance highlights 1`] = `
[
  {
    "evidence": "SOC 2 Type II report and ISO certificates in vendor portal.",
    "framework": "SOC 2 Type II + ISO 27001",
    "highlight": "Shared control inheritance packages accelerate third-party assessments across global teams.",
  },
]
`;

exports[`Industry section view models > snapshots mapped solutions and validates linked titles 1`] = `
[
  {
    "href": "/solutions/mnemosyne/",
    "outcome": "Cuts investigation prep time in half.",
    "positioning": "Centralizes telemetry and governed analytics for financial services teams.",
    "slug": "mnemosyne",
    "title": "Mnemosyne Activation Fabric",
  },
  {
    "href": "/solutions/hermes/",
    "outcome": null,
    "positioning": "Delivers policy-driven automations that pass internal risk reviews without manual rewrites.",
    "slug": "hermes",
    "title": "Hermes Automation Cloud",
  },
]
`;

exports[`Industry section view models > snapshots persona use cases 1`] = `
[
  {
    "automationLevel": "Automated alerts with manual approvals.",
    "narrative": "Blend telemetry, approvals, and audits into a single mission control experience for stakeholders.",
    "persona": "Operations executives",
    "title": "Unified control room",
  },
  {
    "narrative": "Auto-generate attestations, POA&Ms, and regulator-ready audit packages with reusable templates.",
    "persona": "Risk and compliance leads",
    "title": "Evidence automation",
  },
]
`;

exports[`Industry section view models > snapshots pressures and metrics 1`] = `
[
  {
    "description": "Teams lose hours stitching together telemetry across finance, risk, and operations dashboards.",
    "metric": "52% of operators cite duplicate investigations as a top risk.",
    "title": "Siloed telemetry inflates response times",
  },
  {
    "description": "Security teams chase spreadsheet attestations instead of working from shared system-of-record evidence.",
    "metric": "8+ hours burned each week summarizing control status.",
    "title": "Manual attestations delay procurement",
  },
  {
    "description": "Program leads hesitate to ship new flows without explicit policy guardrails baked into orchestrations.",
    "title": "Automation programs stall on compliance reviews",
  },
]
`;

exports[`Industry section view models > snapshots the hero payload 1`] = `
{
  "hero": {
    "copy": "Structured playbooks keep executives aligned on compliance, automation, and go-live velocity.",
    "eyebrow": "Regulated Industries",
    "headline": "Mission ready intelligence",
    "icon": "finance",
  },
  "title": "Finance Intelligence",
}
`;

exports[`Industry section view models > tracks the registered industry icon slugs for schema parity > industry-icon-slugs 1`] = `
[
  "finance",
  "healthcare",
  "public-sector",
  "energy",
  "manufacturing",
  "transport",
  "law",
  "government",
  "military",
  "intelligence",
]
`;
